docs/src/main/sphinx/release/release-438.md

```diff
@@ -0,0 +1,33 @@
+# Release 438 (31 Jan 2024)
+
+## General
+
+* Add support for using `char` and `varchar` types with the {func}`reduce_agg`
```

Comment:
I think this also applies to other types such as `uuid`, `ip_address`, `geometry`, `HyperLogLog`, etc.

Reply:
Yes. If only backed by the Slice